packages/wallet/dapp-client/src/utils/index.ts

```diff
@@ -127,23 +127,23 @@ export const getNetwork = (chainId: number) => {
   return network
 }
 
-export const getRpcUrl = (chainId: number) => {
+export const getRpcUrl = (chainId: Network.ChainId | bigint | number) => {
   const network = getNetwork(chainId)
 
   const url = applyTemplate(NODES_URL, { network: network.name })
 
   return url
 }
 
-export const getRelayerUrl = (chainId: number) => {
+export const getRelayerUrl = (chainId: Network.ChainId | bigint | number) => {
```

Comment:
Not sure if Network.ChainId does anything here since bigint is a broader type. Does this help in intellisense?